src/arrayMethodReduce.js

```diff
@@ -5,7 +5,16 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    const argumentsLengthCompare = arguments.length < 2;
```

Comment:
But it returns `boolean` so I'd call it `hasInitialValue`, in this case when we add `has` we show that this is `boolean` variable